core/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>puppetlabs</groupId>
+    <groupId>puppetlabs.org.jruby</groupId>
     <artifactId>jruby-parent</artifactId>
     <version>1.7.26-puppetlabs-1-SNAPSHOT</version>
```

Comment:
Do you think it would be better to base this off of the last formal release that JRuby upstream did - like `1.7.25-puppetlabs-1-SNAPSHOT`?   If JRuby upstream ends up slipping their 1.7.26 out a ways and includes a bunch of other commits in the interim, just wondering if us having called ours `1.7.26....` would be confusing.  I could go either way on this, just wanted to see what you thought.